articles/service-fabric/service-fabric-get-started-containers.md

```diff
@@ -327,6 +327,56 @@ NtTvlzhk11LIlae/5kjPv95r3lw6DHmV4kXLwiCNlcWPYIWBGIuspwyG+28EWSrHmN7Dt2WqEWqeNQ==
 </ServiceManifestImport>
 ```
 
+### Configure cluster-wide credentials
+
+Starting v6.3, Service Fabric allow user to configure cluster-wide credentials which can be used as default repository credentials by applications.
+
+You can enable/disable the feature by adding a "UseDefaultRepositoryCredentials" attribute to ContainerHostPolicies in ApplicationManifest.xml with a "true/false" Boolean value.
+
+```xml
+<ServiceManifestImport>
+    ...
+    <Policies>
+        <ContainerHostPolicies CodePackageRef="Code" UseDefaultRepositoryCredentials="true">
+            <PortBinding ContainerPort="80" EndpointRef="Guest1TypeEndpoint"/>
+        </ContainerHostPolicies>
+    </Policies>
+    ...
+</ServiceManifestImport>
+```
+
+This will tell Service Fabric to use the default repository credentials which you can specify in the ClusterManifest under the hosting section.  If UseDefaultRepositoryCredentials is set to true, service fabric will now read the following values in clustermanifest:

```

Comment:
What is the ClusterManifestTemplate.json?
When do you edit it?
How do you edit this?

Reply:
I think I need to add following link to the doc
https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-fabric-settings

Reply:
I don't think adding a link to that following doc is enough. Cluster Manifest is a term that the customer isn't exactly familiar with. We should refer to the fabricSettings in the ARM template.